packages/vue/src/pages/crm-contact-list/api/crm-contact-service.ts

```diff
@@ -0,0 +1,8 @@
+import http from '@/api/http';
+
+class CrmContactService {
+     getAll = () =>
```

Comment:
I see that you use different indentations in different places. For axample here 4 spaces and here 2 spaces https://github.com/DevExpress/devextreme-ui-template-gallery/pull/58/files#diff-ba57406a9b57055921f60eaf5caa19f399b9eb0083685bd83e581c9794ef15c2R87